websockets/blob.go

```diff
@@ -0,0 +1,75 @@
+package websockets
+
+import (
+	"bytes"
+
+	"github.com/grafana/sobek"
+
+	"go.k6.io/k6/js/common"
+)
+
+type Blob struct {
```

Comment:
I'm not 100% if we do really want this to remain specific to the extension, or to be available in k6 as another experimental package (similar to `streams`).
cc/ @mstoykov 

To be honest, I developed this within k6 codebase in order to reuse all the WPTs set up, at least while coding. So, I'm fine with any approach. If we prefer to keep it here, I can add a few tests specifically for `Blob` manually.

Reply:
As I said internally - IMO moving this to core *now* will slow down the stabilization of the extension.

Which happens to be the reason we are implementing this to begin with.

I think it will be a lot easier to try to move this to core and usable in other places *after* the extension is out of experimental.

Reply:
Great then, we can move forward as-is - thanks! 🙇🏻